README.md

```diff
@@ -18,7 +18,7 @@ Features:
   * Ability to charge multiple buckets with arbitrary token amounts at once
   * Ability to "reserve" tokens and to create "token debt"
 
-Redis version 3.2 or newer is needed.
```

Comment:
This is actually meant to refer to "Redis, the server", not "Redis, the ruby gem".
So better keep it at `3.2`, I think.

Reply:
I forced-pushed this comment away.